### PR TITLE
Add Home Run Bat (Traitor Item)

### PR DESCRIPTION
**Pull request by angelofallars**
# Description

Adds the home run bat as a new item for traitors, a new traitor weapon that knocks targets back significantly.

The home run bat has the same Nyanocombat stats as the baseball bat, with differences: damage has been reduced by around 25%, same as the baseball bat, and the Stamina damage on power attacks is 25.

Originally made for Delta-V (https://github.com/DeltaV-Station/Delta-v/pull/1476)

## Media

<details>

image
image

</details>

## Changelog

:cl: musicmanvr
- add: Added the Home Run Bat for Syndicate operatives. Send your foes flying!

### CODE REVIEW

**Review by VMSolidus — APPROVED** [review]

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/home_run_bat.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    heavyRateModifier: 0.5
+    heavyDamageBaseModifier: 1.75
+    heavyStaminaCost: 25
+    maxTargets: 2
+    angle: 120
+    soundHit:
+      collection: ExplosionSmall
+  - type: MeleeRequiresWield # You can't hit a home run with one hand, jimbo.
+  - type: MeleeThrowOnHit
+    speed: 30
```
Comment (VMSolidus) [inline Resources/Prototypes/Entities/Objects/Weapons/Melee/home_run_bat.yml:24]: Oh, this datafield doesn't exist. 
```suggestion
    speed: 30
```

**Review by VMSolidus — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/home_run_bat.yml (hunk 1 above)
Comment (angelofallars) [reply Resources/Prototypes/Entities/Objects/Weapons/Melee/home_run_bat.yml:24]: If you are talking about `speed`, the datafield is present in `MeleeThrowOnHitComponent`. https://github.com/Simple-Station/Einstein-Engines/blob/393116752ca73fba533ef404f0e7c0cafc1d6737/Content.Shared/Weapons/Melee/Components/MeleeThrowOnHitComponent.cs#L22

Comment (VMSolidus) [reply Resources/Prototypes/Entities/Objects/Weapons/Melee/home_run_bat.yml:24]: You're right, I clicked the wrong line.

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/home_run_bat.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    lifetime: 5
+  - type: Item
+    size: Large
+  - type: Tool
+    speedModifier: 0.5 # it's very heavy, it rolls slower than a wooden bat
```
Comment (VMSolidus) [inline Resources/Prototypes/Entities/Objects/Weapons/Melee/home_run_bat.yml:29]: That would be this line that is the offending line.

**Review by VMSolidus — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/home_run_bat.yml (hunk 2 above)
Comment (angelofallars) [reply Resources/Prototypes/Entities/Objects/Weapons/Melee/home_run_bat.yml]: Resolved with [f99bcd2](https://github.com/Simple-Station/Einstein-Engines/pull/722/commits/f99bcd2fc5855dcb32276fea013ef16f400bface)

**Review by DangerRevolution — APPROVED** [review]

**Review by VMSolidus — APPROVED** [review]

**Pull request merged**